### ISSUE TITLE
Fix docz

### ISSUE THREAD

**Issue by j8seangel** [GlobalFishingWatch/map-components#44]
```
Module parse failed: Unexpected token (3:11)
You may need an appropriate loader to handle this file type.
| export { default } from './map';
|
> if (typeof default !== 'undefined' && default && default === Object(default)) {
|   Object.defineProperty(default, '__filemeta', {
|     enumerable: true,

 @ ./src/map/map.mdx 17:0-29 188:15-18 196:29-32
 @ ./.docz/app/imports.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/react-dev-utils/webpackHotDevClient.js ./.docz/app/index.jsx
```

Thread here: https://github.com/pedronauck/docz/issues/704 but not fixed yet

**Comment by minheq** [comment]
I am seeing exact same error

**Comment by j8seangel** [comment]
Fixing 0.13.7 exact version in package.json make it works

See https://github.com/pedronauck/docz/issues/704#issuecomment-482139614

**Comment by minheq** [comment]
@j8seangel 
Downgrading 0.13.7 is not a fix for using 1.0.0. I would insist on re-opening this issue given the error we see is different from that of in the https://github.com/pedronauck/docz/issues/704 issue

**Comment by j8seangel** [comment]
I know this is not a fix but as upgrading isn't a priority for us as now we can keep using this version we can mark this as closed and will keep you updated with the research on how to upgrade version. 

We will open a new issue to migrate to v1.0.0 and thanks for the interest!
